### PR TITLE
Add comp infos to userscript page (closes #129, #16)

### LINKED ISSUE #129: Adguard should be friendly to userscripts developers
Here it is:
https://github.com/AdguardTeam/AdguardForWindows/issues/1218

### PR DESCRIPTION
**Pull request by Bluscream**
This is still work in progress but really needs to be done to help people figure out problems with their scripts

Source: https://github.com/AdguardTeam/AdguardForWindows/issues/1218

### CODE REVIEW

File: 02.general/06.userscripts/docs.en.md [hunk 1]
```diff
@@ -67,4 +71,117 @@ If you like the idea of personifying your browser with the help of userscripts,
...
+@run-at - document-idle option is not supported.
+```
+
+##### Unsupported properties
+These properties will be simply ignored by Adguard.
+```
+@namespace
+@unwrap
+@noframes - Adguard won't execute a userscript inside of any iframe.
+@grant - Everything is granted by default.
```
Comment (ameshkov) [inline 02.general/06.userscripts/docs.en.md]: It is actually supported:)

File: 02.general/06.userscripts/docs.en.md [hunk 2]
```diff
@@ -67,4 +71,117 @@ If you like the idea of personifying your browser with the help of userscripts,
...
+@connect - Adguard uses `@grant host:site.com` instead!
```
Comment (ameshkov) [inline 02.general/06.userscripts/docs.en.md]: This one is also supported

File: 02.general/06.userscripts/docs.en.md [hunk 3]
```diff
@@ -67,4 +71,117 @@ If you like the idea of personifying your browser with the help of userscripts,
...
+
+##### Partially supported properties
+```
+@run-at - document-idle option is not supported.
+```
+
+##### Unsupported properties
+These properties will be simply ignored by Adguard.
+```
+@namespace
```
Comment (ameshkov) [inline 02.general/06.userscripts/docs.en.md]: This one is supported

**Comment by ameshkov** [comment]
This is a good idea, we should use this PR as a base for the documentation. Plz give me some time

**Review by Dariamag — DISMISSED** [review]

**Review by Dariamag — APPROVED** [review]

**Review by Dariamag — APPROVED** [review]

**Comment by Dariamag** [comment]
There are some conflicts that for some reason can't be resolved and that's why merged. 
I believe the reason might be the changed structure of KB (we don't have _02.general_ anymore)

@Bluscream thanks! I've added your changes along with @sfionov changes to a new pull request https://github.com/AdguardTeam/AdguardKnowledgeBase/pull/610

**Comment by Bluscream** [comment]
:)

**Pull request closed without merging**